hasjob/templates/macros.html.jinja2

```diff
@@ -199,20 +199,15 @@
   {{ campaign_header_postscript() }}
 {%- endmacro -%}
 
-{%- macro filters_setup_script(job_filters) -%}
+{%- macro filters_setup_script(job_filters, selected_filters) -%}
   {% with all_jobs_filters=job_filters() %}
     <script type="text/javascript">
-      window.Hasjob.Config.jobsArchive = {{request.args.get('archive') | tojson}};
-      window.Hasjob.Config.jobLocationFilters = {{all_jobs_filters.job_location_filters | tojson}};
-      window.Hasjob.Config.jobTypeFilters = {{all_jobs_filters.job_type_filters | tojson}};
-      window.Hasjob.Config.jobCategoryFilters = {{all_jobs_filters.job_category_filters | tojson}};
-      window.Hasjob.Config.selectedLocations = {{request.args.getlist('l') | tojson}};
-      window.Hasjob.Config.selectedTypes = {{request.args.getlist('t') | tojson}};
-      window.Hasjob.Config.selectedCategories = {{request.args.getlist('c') | tojson}};
-      window.Hasjob.Config.selectedQuery = {{request.args.getlist('q') | tojson}};
-      window.Hasjob.Config.selectedCurrency = {{request.args.get('currency') | tojson}};
-      window.Hasjob.Config.pay = {{request.args.get('pay') | tojson}};
-      window.Hasjob.Config.equity = {{request.args.get('equity') | tojson}};
+      window.Hasjob.Config.allFilters = {{all_jobs_filters | tojson}};
+      {%- if selected_filters %}
```

Comment:
`selected_filters` is a mandatory parameter to this macro. Did you mean to specify a default value above?